assets/constants.ts

```diff
@@ -51,6 +51,7 @@ export default {
   "DEFAULT_COLOR": "#a33f40",
   "DEFAULT_THUMBNAIL_ICON": "Lightbulb",
   "DEFAULT_CATEGORY_ICON": "Lightbulb",
+  "DEFAULT_SKILL_IDS_FOR_DIAGNOSTIC_TEST": [],
```

Comment:
I don't think you need this. It's fine for the default to "naturally" be an empty list; there is no skill that would by default fit into all topics.

Reply:
Updated.